src/js_schema.hpp

```diff
@@ -169,6 +174,53 @@ static inline void parse_property_type(StringData object_name, Property& prop, S
             prop.type |= PropertyType::Object | PropertyType::Array;
         }
     }
+    else if (type == "set") {
+        if (prop.object_type == "bool") {
+            prop.type |= PropertyType::Bool | PropertyType::Set;
+            prop.object_type = "";
+        }
+        else if (prop.object_type == "int") {
+            prop.type |= PropertyType::Int | PropertyType::Set;
+            prop.object_type = "";
+        }
+        else if (prop.object_type == "float") {
+            prop.type |= PropertyType::Float | PropertyType::Set;
+            prop.object_type = "";
+        }
+        else if (prop.object_type == "double") {
+            prop.type |= PropertyType::Double | PropertyType::Set;
+            prop.object_type = "";
+        }
+        else if (prop.object_type == "string") {
+            prop.type |= PropertyType::String | PropertyType::Set;
+            prop.object_type = "";
+        }
+        else if (prop.object_type == "date") {
+            prop.type |= PropertyType::Date | PropertyType::Set;
+            prop.object_type = "";
+        }
+        else if (prop.object_type == "data") {
+            prop.type |= PropertyType::Data | PropertyType::Set;
+            prop.object_type = "";
+        }
+        else if (prop.object_type == "decimal128") {
+            prop.type |= PropertyType::Decimal | PropertyType::Set;
+            prop.object_type = "";
+        }
+        else if (prop.object_type == "objectId") {
+            prop.type |= PropertyType::ObjectId | PropertyType::Set;
+            prop.object_type = "";
+        }
```

Comment:
UUID has been merged to `develop` so it should be possible to add it.

Reply:
The schema parsing is fixed.  Will add unit tests